model/textparse/openmetricsparse.go

```diff
@@ -219,6 +219,13 @@ func (p *OpenMetricsParser) nextToken() token {
 	return tok
 }
 
+func (p *OpenMetricsParser) ParseError(err error) error {
```

Comment:
Also, there's already a function named `parseError`. For the sake of clarity, maybe this one should be renamed  like `ErrorPosition` or something along those lines?